src/kudu/consensus/raft_consensus.cc

```diff
@@ -192,7 +192,8 @@ RaftConsensus::RaftConsensus(
     RaftPeerPB local_peer_pb,
     scoped_refptr<ConsensusMetadataManager> cmeta_manager,
     ThreadPool* raft_pool,
-    ElectionDecisionCallback edcb)
+    ElectionDecisionCallback edcb,
+    TermAdvancementCallback tacb)
```

Comment:
the constructor wit get unwieldy as we add more things. Is it possible to
use Set functions instead, because these callbacks are optional.